### ISSUE TITLE
Regarding the hyperparameters weight decay and teacher temperature 

### ISSUE THREAD

**Issue by bluenights** [facebookresearch/dino#181]
Thanks for the great work. Could you please share the rationale behind the choices of two hyperparameters,  weight decay with a cosine schedule , and teacher temperature with warmup?
